commands/user_e2e_test.go

```diff
@@ -136,3 +136,72 @@ func (s *MmctlE2ETestSuite) TestSearchUserCmd() {
 		s.Equal("Unable to find user '"+emailArg+"'", printer.GetErrorLines()[0])
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestCreateUserCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	s.RunForAllClients("Don't create new user w/o username", func(c client.Client) {
+		printer.Clean()
+		cmd := &cobra.Command{}
+
+		err := userCreateCmdF(c, cmd, []string{})
+		s.EqualError(err, "Username is required: flag accessed but not defined: username")
```

Comment:
Let's also test from application interface perspective.
```suggestion
		s.EqualError(err, "Username is required: flag accessed but not defined: username")
		s.Require().Empty(printer.GetLines())
```